doc/Comments.xml

```diff
@@ -556,8 +575,9 @@ a function, operation, etc., it is possible to group them into suitable chunks.
 This can be particularly useful if there are several definitions of an operation
 with several different argument types, all doing more or less the same to the arguments.
 Then their manual items can be grouped, sharing the same description and return type information.
-Note that it is currently not possible to give a header to the Group in the manual,
-but the generated ManItem heading of the first entry will be used.
+You can give a heading to the Group in the manual with the <C>@GroupTitle</C>
+command; if that is not supplied, then the generated ManItem heading of the
```

Comment:
`ManItem` is also bad here, and should not be used -- but of course it was here before, so I do not expect you to fix it (rather, I expect me or @sebasguts to fix it)

Reply:
Why do you think this is bad? `ManItem` is a special kind of subsection in GAPDoc, which does not need to have a heading, but will create one. What would you suggest?

Reply:
First off, it should at the very least be typeset with `<C>` or so -- better would be a Ref to the relevant GAPDoc manual entry, though.

But really, why is a technical term from GAPDoc used to explain things here? In order to understand this, now one needs to be familiar with both AutoDoc and GAPDoc, *and* how the conversion from one to the other works exactly.

It would be much better if the AutoDoc explanation was self-contained. Perhaps in the future, we do not just output GAPDoc, but also some other format?

At the same time, it can be useful to document how certain AutoDoc constructs translate to GAPDoc. But IMHO this should be separate (say, in a separate paragraph). And then for more (ideally all) AutoDoc, not just a random (?) aubset

Reply:
Yep, you are right, it is probably bad to refer to GAPDoc internals (probably out of laziness here). Anyway, as you stated, I should probably fix this myself, so this should not block this PR.